src/internal/operators/audit.ts

```diff
@@ -1,7 +1,7 @@
 import { Subscriber } from '../Subscriber';
 import { MonoTypeOperatorFunction, ObservableInput } from '../types';
 
-import { operate } from '../util/lift';
```

Comment:
The `operate` function is being removed everywhere.

Look for a conversion from:  `operate(fn)` to `(source) => new Observable(subscriber => fn(source, subscriber))` (basically)